src/pages/SignIn/SignIn.jsx

```diff
@@ -1,10 +1,39 @@
 import React from 'react'
-
+import { Link } from 'react-router-dom'
+import {NavBar,Footer} from '../HomePage/HomePageComponents'
+import '../SignIn/SignIn.css'
 function SignIn() {
 return (
 <div>
-    <h1>Blank SignIn page for Route checking</h1>
+    <NavBar />
+    {/* <h1>Blank SignIn page for Route checking</h1> */}
+    <div className="login-container">
+        <h1 className="header-large">LOG IN</h1>
+        <div className="input">
```

Comment:
Have a unique classNames. This might mislead.

Reply:
Sure, Will update them 👍️